admin/conf/routes

```diff
@@ -10,6 +10,7 @@ GET     /jobs/:group/:name            com.lucidchart.piezo.admin.controllers.Job
 POST    /jobs/:group/:name            com.lucidchart.piezo.admin.controllers.Jobs.putJob(group: String, name: String)
 DELETE  /jobs/:group/:name            com.lucidchart.piezo.admin.controllers.Jobs.deleteJob(group: String, name: String)
 GET     /jobs/:group/:name/editor     com.lucidchart.piezo.admin.controllers.Jobs.getEditJob(group: String, name: String)
+GET     /jobs/:group/:name/duplicator com.lucidchart.piezo.admin.controllers.Jobs.getDuplicateJob(group: String, name: String)
```

Comment:
Since we're returning the same view I wonder if it makes sense to model it as the same resouce as /job/:group/:name/editor with a query string of ?template=foo